cdc/puller/puller.go

```diff
@@ -133,10 +131,7 @@ func (p *pullerImpl) Run(ctx context.Context) error {
 	tableID, tableName := util.TableIDFromCtx(ctx)
 	metricOutputChanSize := outputChanSizeGauge.WithLabelValues(captureAddr, changefeedID, tableName)
 	metricEventChanSize := eventChanSizeGauge.WithLabelValues(captureAddr, changefeedID, tableName)
-	metricMemBufferSize := memBufferSizeGauge.WithLabelValues(captureAddr, changefeedID, tableName)
 	metricPullerResolvedTs := pullerResolvedTsGauge.WithLabelValues(captureAddr, changefeedID, tableName)
-	metricEventCounterKv := kvEventCounter.WithLabelValues(captureAddr, changefeedID, "kv")
-	metricEventCounterResolved := kvEventCounter.WithLabelValues(captureAddr, changefeedID, "resolved")
```

Comment:
Are `memBufferSizeGauge` `kvEventCounter` and `kvEventCounter` dead code?

Please also update grafana dashboard and metrics doc.

Reply:
ok

Reply:
I will refine metrics in another PR

Reply:
https://github.com/pingcap/ticdc/pull/1584